panel/pane/vtk.py

```diff
@@ -0,0 +1,519 @@
+# coding: utf-8
```

Comment:
This file has quite a bit of code; was it really all developed from scratch for Panel?  Or was it adapted from some other starting point?  If the latter, probably need to include the license of the original code.

Reply:
the code is an adaptation from this Macro
https://raw.githubusercontent.com/Kitware/vtk-js/master/Utilities/ParaView/export-scene-macro.py
It allows to convert a paraview scene to a zip file

Most of developpement are inspired from
https://kitware.github.io/vtk-js/examples/SceneExplorer.html

I'm not very aware about licensing

Reply:
Ok, I think you'd need to paste in the license from: https://github.com/Kitware/vtk-js/blob/master/LICENSE

Reply:
(And should include the link to the original example, for reference.)